rclpy/executors/examples_rclpy_executors/publish_pointcloud.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Evan Flynn
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ctypes
```

Comment:
Especially for an example, do we really have to use ctypes to create the string buffer?  Is there something more idiomatic that we can use?